plugin/trino-hive-hadoop2/bin/common.sh

```diff
@@ -211,3 +211,14 @@ function deploy_core_site_xml() {
         'sed "${@:2}" "/docker/files/$1" > /etc/hadoop/conf/core-site.xml' \
         bash "$template" "${args[@]}"
 }
+
+# Checks if Gitflow Incremental Builder (GIB) is enabled and the trino-hive-hadoop2 module should be build and/or tested
+function abort_if_not_gib_impacted() {
+    local module=plugin/trino-hive-hadoop2
+    local impacted_log=gib-impacted.log
+    if [ -f "$impacted_log" ] && ! grep -q "^$module$" "$impacted_log"; then
```

Comment:
nit: for readability

```suggestion
    if [ -f "$impacted_log" ] && ! grep -q "^${module}$" "$impacted_log"; then
```

Reply:
Done, thanks!